ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -63,30 +84,65 @@ open class Tooltip: NSObject {
     private var gestureView: UIView?
     private var dismissMode: DismissMode = .tapAnywhere
 
-    private override init() { }
+    private override init() {
```

Comment:
`    /// The alignment of the text in the tooltip. Defaults to natural alignment (left for LTR languages, right for RTL languages).
    @objc public var textAlignment: NSTextAlignment = .natural`

@mischreiber : to text alignment make sense as a token or as a property?